corehq/blobs/migrate.py

```diff
@@ -650,74 +650,6 @@ def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100, limit
 
 
 MIGRATIONS = {m.slug: m for m in [
-    Migrator('domains', [Domain], CouchAttachmentMigrator),
-    Migrator('commcare_builds', [CommCareBuild], CouchAttachmentMigrator),
-    Migrator('invoice_pdfs', [InvoicePdf], CouchAttachmentMigrator),
-    Migrator("saved_exports", [SavedBasicExport], CouchAttachmentMigrator),
-    Migrator("applications", [
-        apps.Application,
-        apps.RemoteApp,
-        ("Application-Deleted", apps.Application),
-        ("RemoteApp-Deleted", apps.RemoteApp),
-    ], CouchAttachmentMigrator),
-    Migrator("multimedia", [
-        hqmedia.CommCareAudio,
-        hqmedia.CommCareImage,
-        hqmedia.CommCareVideo,
-        hqmedia.CommCareMultimedia,
-    ], CouchAttachmentMigrator),
-    Migrator("xforms", [
-        xform.XFormInstance,
-        ("XFormInstance-Deleted", xform.XFormInstance),
-        xform.XFormArchived,
-        xform.XFormDeprecated,
-        xform.XFormDuplicate,
-        xform.XFormError,
-        xform.SubmissionErrorLog,
-        ("HQSubmission", xform.XFormInstance),
-    ], CouchAttachmentMigrator),
-    Migrator("cases", [
-        cases.CommCareCase,
-        ('CommCareCase-deleted', cases.CommCareCase),
-        ('CommCareCase-Deleted', cases.CommCareCase),
-        ('CommCareCase-Deleted-Deleted', cases.CommCareCase),
-    ], CouchAttachmentMigrator),
-    MultiDbMigrator("migrate_backend",
```

Comment:
we still need this one, but it will be changing with my work on blob metadata

Reply:
Added back